staging/src/k8s.io/component-base/metrics/buckets.go

```diff
@@ -17,12 +17,34 @@ limitations under the License.
 package metrics
 
 import (
+	"time"
+
 	"github.com/prometheus/client_golang/prometheus"
 )
 
 // DefBuckets is a wrapper for prometheus.DefBuckets
 var DefBuckets = prometheus.DefBuckets
 
+// DefNativeHistogramBucketFactor defines a sane default for the Prometheus
+// native histogram exponential bucket factor. The value of 1.1 means each
+// bucket is 10% wider than the previous bucket.
+//
+// For more information on Prometheus Native Histograms, see the upstream
+// client_golang documentation.
+//
+// https://pkg.go.dev/github.com/prometheus/client_golang/prometheus#HistogramOpts
+var DefNativeHistogramBucketFactor = 1.1
+
+// DefNativeHistogramMaxBucketNumber defines a sane default for the Prometheus
+// native histogram limit to the number of sparse buckets. This avoids
+// unbounded memory requirements for native histogram tracking.
+var DefNativeHistogramMaxBucketNumber = 200
```

Comment:
Maybe let's use 160 because this coincides with the default max buckets of the OTel exponential histograms?

Reply:
Sure, that seems OK to me.